values.md

```diff
@@ -1,27 +1,25 @@
 # Kubernetes Community Values
 
-Kubernetes Community culture is frequently cited as a substantial contributor to the meteoric rise of this Open Source project.  Below are the distilled values which have evolved over the last many years in our community pushing our project and peers toward constant improvement.
+Kubernetes Community culture contributes substantially to the project's success. The following values have evolved over time, pushing our project and peers toward constant improvement.
 
 ## Distribution is better than centralization
 
-The scale of the Kubernetes project is only viable through high-trust and high-visibility distribution of work, which includes delegation of authority, decision making, technical design, code ownership, and documentation.  Distributed asynchronous ownership, collaboration, communication and decision making are the cornerstone of our world-wide community.
+The scale of the Kubernetes project is only viable through high-trust and high-visibility distribution of work, which includes delegation of authority, decision making, technical design, code ownership, and documentation.  Distributed asynchronous ownership, collaboration, communication and decision making are the cornerstones of our world-wide community.
```

Comment:
non-blocking nit: I agree the verb and pluralization should agree, but would "is the cornerstone" read better? I bumped on the plural "cornerstones".

I _think_ "Distributed $stuff is the cornerstone..." is still coherent, but I'll defer to the docs folks.

Reply:
@liggitt 

> would "is the cornerstone" read better?

For what it's worth, I think neither of them read especially well. :shrug: I'd like to use different language altogether, but I wanted to scope both the originating issue and any implementation reviews conservatively.

For purposes of this PR, I think either is/singular or are/plural can work.